workload_mapper/rails/clue.rb

```diff
@@ -0,0 +1,17 @@
+if system.runs_service?("apache2") && system.has_file?("/usr/bin/rails")
+  apache_vhosts = system["unmanaged_files"].files.select { |file| /vhosts\.d/ =~ file.name }.map { |file| file.name }
```

Comment:
Line is too long. [117/100]<br>Pass `&:name` as an argument to `map` instead of a block.